arroyo-worker/src/connectors/kafka/source/mod.rs

```diff
@@ -272,11 +272,20 @@ where
                                 let iter = self.deserializer.deserialize_slice(v).await;
 
                                 for value in iter {
-                                    ctx.collector.collect(Record {
-                                        timestamp: from_millis(timestamp as u64),
-                                        key: None,
-                                        value: value?,
-                                    }).await;
+                                    match value {
```

Comment:
Does this need to be updated to use collect_source_record?

Reply:
Yup, missed this. It's done now.